service-loadbalancer/README.md

```diff
@@ -15,6 +15,35 @@ This module will not help with the latter
 
 ## Overview
 
+The image below roughly describes the service-loadbalancer alternative (compared with using NodePorts).
```

Comment:
This sentence is confusing. Suggest removing and moving this diagram down to the section where we discuss tcp (since it's actually tcp specific).